lib/generate/generate.go

```diff
@@ -16,6 +18,22 @@ func WithSpecialChars(wordlist []string) []string {
 	return wordlist
 }
 
+func WithNumbers(wordlist []string, specificRange int) []string {
+	Numbers := make([]int, specificRange+1)
+	for _, word := range wordlist {
+		for i := range Numbers {
+			var newPassword = word + strconv.Itoa(i)
+
+			for utils.Contains(wordlist, newPassword) {
+				newPassword = word + strconv.Itoa(i)
+			}
```

Comment:
I think this function is no longer useful here ?

Reply:
I think so too, and also there is a similar one in upper version of go, i will delete it !